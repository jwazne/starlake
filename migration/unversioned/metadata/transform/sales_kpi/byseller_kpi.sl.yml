task:
  parseSQL: true
  sink:
    partition:
      - timestamp