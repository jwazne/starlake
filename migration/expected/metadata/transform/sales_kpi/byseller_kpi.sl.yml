---
version: 1
task:
  parseSQL: true
  sink:
    partition:
      - timestamp